src/profile/thetapr.jl

```diff
@@ -28,12 +28,13 @@ function profileθj!(
 ) where {T}
     (; m, fwd, rev) = val
     optsum = m.optsum
-    (; final, fmin, lowerbd) = optsum
+    (; final, fmin) = optsum
     j = parsej(sym)
     θ = copy(final)
-    lbj = lowerbd[j]
     osj = optsum
     opt = Opt(osj)
+    pmj = m.parmap[j]
+    lbj = pmj[2] == pmj[3] ? zero(T) : T(-Inf)
```

Comment:
this pattern has come up enough, I wonder if it makes sense to define a helper function `_lowerbound(parmapj)`

[noblock]